### ISSUE TITLE
Error when setting sortByKey (Kotlin)

### ISSUE THREAD

**Issue by jlee-everlaw** [littlerobots/version-catalog-update-plugin#27]
Hello, this might be a newbie question, but I get the following error
```
* What went wrong:
Script compilation error:

  Line 7:     sortByKey.set(false)
                        ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                            public inline operator fun kotlin.text.StringBuilder /* = java.lang.StringBuilder */.set(index: Int, value: Char): Unit defined in kotlin.text
```

My build.gradle.kts looks like
```
plugins {
    alias(libs.plugins.gradleVersions)
    alias(libs.plugins.versionCatalogUpdater)
}

versionCatalogUpdate {
    sortByKey.set(false)
}

...
```

**Comment by hvisser** [comment]
Yes this has been fixed for the next version, but not yet released (see #26), in the meantime using `sortByKey = false` should work.

**Comment by hvisser** [comment]
I've released version 0.3.1, it should be on Maven Central soon.
